src/aspire/utils/indices.py

```diff
@@ -0,0 +1,36 @@
+"""
+Utility functions for common index arithmetic.
+"""
+
+import numpy as np
+import random
+
+def random_tri_indices(nsamples, n, k=0, m=None,
```

Comment:
Generally I prefer to use longer names for parameters such as  `n_row` or `n_col`.

Reply:
The names come from `numpy`. They know what they are doing.